src/main/java/io/rocketpartners/cloud/action/sql/SqlDb.java

```diff
@@ -175,7 +175,15 @@ protected void shutdown0()
       //      }
 
       //      String dbname = (String) req.getChain().get("db");
-      SqlDb db = (SqlDb) table.getDb();
+      SqlDb db = null;
+      if (table == null)
+      {
+         db = this;
+      }
+      else
+      {
+         db = (SqlDb) table.getDb();
+      }
```

Comment:
table is null in RestGetAction line 278. seemed reasonable to me that if we're in SqlDb and we don't have a table, we can assume we're calling on ourself.